src/core/java/com/enderio/core/common/util/PlayerInteractionUtil.java

```diff
@@ -24,4 +24,19 @@ public static void putStacksInInventoryFromWorldInteraction(Player player, Block
             }
         }
     }
+
+    /**
+     * Tries to transfer a single ItemStack, which was the result of a world interaction, into the player's inventory, and if not successful,
+     * drops them on the ground where the event took place.
+     *
+     * @param player - the player that performed the event.
+     * @param interactionPosition - Where the interaction took place, i.e. where the player clicked, the block was etc.
+     * @param item - The ItemStack that comes from the event.
+     */
+    public static void putItemInInventoryFromWorldInteraction(Player player, BlockPos interactionPosition, ItemStack item){
```

Comment:
can you redirect the method that adds a list to call this method for each itemstack in the list?